automation/check-patch.e2e.sh

```diff
@@ -17,8 +17,8 @@ main() {
     source automation/setup.sh
     cd ${TMP_PROJECT_PATH}
 
-    echo 'Run golint'
-    make lint
+    echo 'Run format'
+    make format
```

Comment:
This target does not check anything though, does it? The original target would fail the check if the code is not formatted properly, this one would just silently pass IIUIC

Reply:
Added `make check`